model/task/db.go

```diff
@@ -1034,6 +1034,36 @@ func FindOneIdNewOrOld(id string) (*Task, error) {
 	return task, err
 }
 
+func MakeOldID(taskID string, execution int) string {
+	return fmt.Sprintf("%s_%d", taskID, execution)
+}
+
+func FindAllNewOrOld(q db.Q, execution int) ([]Task, error) {
```

Comment:
It seems weird to have the execution be a parameter, since I don't think you'd ever pass anything other than 0. Can this be something like a FindAllFirstExecution?

Reply:
done